tools/crates-validator/src/bin/main.rs

```diff
@@ -14,18 +14,18 @@
 
 use std::{fs::File, io::BufReader, path::PathBuf};
 
-use anyhow::{Context, Result};
+use anyhow::{bail, Context, Result};
 use clap::Parser;
 use colored::Colorize;
-use risc0_crates_validator::{ProfileConfig, RunStatus, ValidatorBuilder};
+use risc0_crates_validator::{types::repo::Repo, ProfileConfig, RunStatus, ValidatorBuilder};
 use tracing::info;
 use tracing_subscriber::EnvFilter;
 
 #[derive(Parser, Debug)]
 #[command(author, version, about, long_about = None)]
 struct Args {
     /// Path to json version of [ProfileConfig]
-    #[arg(short = 'p', long, default_value = "./profiles/primary.json")]
+    #[arg(short = 'P', long, default_value = "./profiles/primary.json")]
```

Comment:
Should change this to ./profiles/config.yaml.

Consider removing this flag from CI job?